standard/namespaces.md

```diff
@@ -112,16 +112,28 @@ The scope of an *extern_alias_directive* extends over the *using_directive*s, *g
 
 Within a compilation unit or namespace body that contains an *extern_alias_directive*, the identifier introduced by the *extern_alias_directive* can be used to reference the aliased namespace. It is a compile-time error for the *identifier* to be the word `global`.
 
-Within C# source code, a type is declared a member of a single namespace. However, a namespace hierarchy referenced by an extern alias may contain types that are also members of other namespaces. For example, if `A` and `B` are extern aliases, the names `A::X`,`B::C.Y` and `global::D.Z` may, depending on the external specification supported by the particular compiler, all refer to the same type.
-
 The alias introduced by an *extern_alias_directive* is very similar to the alias introduced by a *using_alias_directive*. See [§13.5.2](namespaces.md#1352-using-alias-directives) for more detailed discussion of *extern_alias_directive*s and *using_alias_directive*s.
 
 `alias` is a contextual keyword ([§6.4.4](lexical-structure.md#644-keywords)) and only has special meaning when it immediately follows the `extern` keyword in an *extern_alias_directive*.
 
-> *Example*: In fact an extern alias could use the identifier `alias` as its name:
+An error occurs if a program declares an extern alias for which no external definition is provided.
+
+> *Example*: The following program declares and uses two extern aliases, `X` and `Y`, each of which represent the root of a distinct namespace hierarchy:
```

Comment:
Would this help clarify the example? I wonder if we'll remember why this exists when we look at it later. (I know I keep forgetting).

```suggestion
> *Example*: The following program declares and uses two extern aliases, `X` and `Y`, each of which represent the root of a distinct namespace hierarchy. The external definition maps the `X` and `Y` namespaces to the assemblies that contain the distinct types with matching fully qualified names:
```

Reply:
I'm not sure... that feels like it's moving away from the generalized view of extern aliases to something specific. On the other hand, it *is* only an example. @Nigel-Ecma, thoughts?

Reply:
I think I agree with @jskeet here. @BillWagner's suggestion makes sense if we're standardising the Roslyn copier's external mechanism that presents an assembly as a namespace that encloses all the namespaces in the assembly; but if we're doing that it has to be specified normatively somewhere.

Maybe we should standardise the external mechanism, or at least as much as what is allowed to be aliased (e.g. assemblies and not individual namespaces within them), but that is probably too big a decision to make as this stage for v6 and is a (theoretical) breaking change.

TL;DR let's fix as per this PR and revisit in v7